build-tools/packages/version-tools/test/internalVersionScheme.test.ts

```diff
@@ -174,21 +174,25 @@ describe("internalScheme", () => {
             assert.isFalse(semver.satisfies(`2.0.0-internal.3.1.0`, range));
         });
 
-        // Skipped for now because they are known to fail. We'll enable them once we've determined how to number our PR
-        // builds
-        it.skip("Prerelease versions do not satisfy ranges", () => {
+        it("Prerelease/dev versions do not satisfy ranges", () => {
             assert.isFalse(
                 semver.satisfies(
-                    `2.0.0-internal.1.1.1.95400`,
+                    `2.0.0-dev.1.1.1.95400`,
```

Comment:
prerelease/dev mean the same? Is this the way we name prerelease versions?

Reply:
Yeah -- only "real" internal releases will get the internal identifier. Everything else is dev. I'll add some comments.